Makefile.am

```
@@ -22,6 +22,7 @@ if ENABLE_STATIC
 bzip3_SOURCES += $(libbzip3_la_SOURCES)
 else
 bzip3_LDADD = libbzip3.la
+bzip3_LDFLAGS = -Wl,-rpath -Wl,$(DESTDIR)$(libdir)
```

I'm pretty sure this will break most distro packaging where `$DESTDIR` is a staging area where files are collected to put in the package. For people installing directly to their system from source it should default to an empty value (effectively `/`) but for distro packaging it will be some temporary target path. The result will be unusable.

If anything you'd want something like this:

```suggestion
bzip3_LDFLAGS = -Wl,-rpath -Wl,$(libdir)
```

But it would be nice to find some precedent. Something still feels a still off to me. A system that doesn't have a path set by default probably shouldn't be defaulting to using that path in the first place.

---

So we're supposed to move the default bzip3 installation directory from `/usr/local/lib` elsewhere?

---

No, `/usr/local` is the right prefix for software installed directly from sources by and end user, distro packagers then override it to be `/usr`. Pretty standard.

Also it looks like this operation is okay. I did some testing and it's just appending anyway. This will also be responsive to being configured with `./configure --prefix=foo` so cover other use cases anyway (such as `/opt`).

As a side note I'm still surprised this default of using `$(libdir)` isn't baked into the default LDFLAGS, but a bit of searching suggests we [aren't the first](https://grep.app/search?current=2&q=-rpath%20-Wl&filter[lang][0]=Makefile) to hit this issue. I'm surprised at the 102 hits and not 10,200 — and even at that many of the 102 hits are [clearly just stabbing in the dark with no clue how empty Makefile variables are supposed to be used](https://github.com/OPAE/intel-fpga-bbb/blob/master/BBB_cci_gemm/base/sw/Makefile#L29-L42) (among other crimes). But none of this is to suggest not merging this, it does solve a problem and doesn't seem to cause more so unless somebody has a better way to suggest I think this is fine.